src/BookThumbnail.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import bookThumbnailCss from './css/bookThumbnail.css';
+import { Link } from 'react-router-dom';
+
+class BookThumbnail extends React.Component {
+  constructor(){
+    super()
+    this.state = {
+      loaded: true
+    }
+  }
+  updateOnLoad(){
```

Comment:
except for the render function declare the rest using arrow functions to avoid the `.bind.this`

in this case it would be:

```
updateOnLoad = () => {
  this.setState({loaded: false})
}

Reply:
why do you need this?